python/sglang/srt/models/qwen2_moe.py

```diff
@@ -368,7 +366,6 @@ def __init__(
             config.vocab_size, config.hidden_size, quant_config=quant_config
         )
         self.logits_processor = LogitsProcessor(config)
-        self.sampler = Sampler()
```

Comment:
just curious, why is this change part of this PR?

Reply:
We don't use vllm's sampler in our code base, this is a mistake by the qwen's contributor.